PresentScreenings/Views/ColorView.cs

```diff
@@ -36,16 +36,25 @@ public static class ColorView
         static readonly Dictionary<ScreeningInfo.ScreeningStatus, NSColor> BgColorByScreeningStatus;
         static readonly Dictionary<ScreeningInfo.TicketsStatus, NSColor> ColorByTicketStatus;
         static readonly NSColor ClickPadColorBlue = NSColor.FromRgba(0, 0, 255, 207);
+        static readonly NSColor ClickPadColorGreen = NSColor.FromRgba(0, 127, 0, 207);
         static readonly NSColor ClickPadColorGrey = NSColor.FromRgba(127, 127, 127, 119);
+        static readonly NSColor ClickPadColorPeach = NSColor.FromRgba(255, 217, 176, 207);
         static readonly NSColor ClickPadTextColorSelected = NSColor.White;
         static readonly NSColor ClickPadTextColorUnselected = NSColor.Red;
+        static readonly Dictionary<Tuple<bool, bool>, NSColor> clickPadColorByVodSelected;
         static readonly NSColor soldOutColorSelected = NSColor.FromRgb(219, 176, 38);
         static readonly NSColor soldOutColorUnselected = NSColor.FromRgb(176, 79, 38);
         #endregion
 
         #region Constructors
         static ColorView()
         {
+            clickPadColorByVodSelected = new Dictionary<Tuple<bool, bool>, NSColor> { };
+            clickPadColorByVodSelected[new Tuple<bool, bool>(false, false)] = ClickPadColorGrey;
+            clickPadColorByVodSelected[new Tuple<bool, bool>(false, true)] = ClickPadColorBlue;
+            clickPadColorByVodSelected[new Tuple<bool, bool>(true, false)] = ClickPadColorPeach;
+            clickPadColorByVodSelected[new Tuple<bool, bool>(true, true)] = ClickPadColorGreen;
```

Comment:
Please consider to define these colors as properties of `Screening` and `OnDemandScreening`.

Reply:
Properties alone won't work.
To support them by virtual/override methods shoots far beyond the target.